utests/tests/ptf/test_testutils.py

```diff
@@ -0,0 +1,6 @@
+from ptf.testutils import simple_igmp_packet
+
+
+def test_simple_igmp_packet__proper_setting_mrtime_mrcode():
+    simple_packet = simple_igmp_packet(igmp_mrtime=10)
+    assert simple_packet["IGMP"].mrcode == 10
```

Comment:
I don't see this test being run in CI by https://github.com/p4lang/ptf/blob/de2b4b3f6467186be4473208737fea67848e3840/.github/workflows/build.yml#L60

Maybe the command needs to be modified so that nose2 can run the tests in the new subdirectory.

Reply:
Actual, this test is run by the Makefile's command `test` https://github.com/p4lang/ptf/pull/158/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R17
I planned to do some refactor around all unittests, then I run it into CI, with separate PR.